### ISSUE TITLE
Rule Proposal: `no-import-reserved-word`

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#4702]
In the spec, the following code is valid:

```js
import {class, default, var} from "foo";
```

Because reserved words are allowed there.
But we cannot use those variables.

```js
import {class, default, var} from "foo";

console.log(class); // syntax error!
```

So this rule reports import declarations which are defining variables of reserved words.

```js
// Valid:
import {a, b, c} from "foo";
import {class as a, default as b, var as c} from "foo";

// Invalid:
import {class, default, var} from "foo";  /*error "class" is a reserved word.*/
                                          /*error "default" is a reserved word.*/
                                          /*error "var" is a reserved word.*/
```

**Comment by michaelficarra** [comment]
Those are not valid. Look again.

**Comment by mysticatea** [comment]
Oh, you're right.
So, this is parser's bug.
